app/models/secret_storage.rb

```diff
@@ -50,6 +50,43 @@ def self.keys
     end
   end
 
+  class HashicorpVault
+
+    SECRET_BACKEND ='secret/'.freeze
```

Comment:
ahh ... should we still prefix samson- or something like that to not mess up the root namespace for others ?